libs/langchain/langchain/schema/runnable/locals.py

```diff
@@ -0,0 +1,156 @@
+from __future__ import annotations
+
+from typing import Any, AsyncIterator, Dict, Iterator, Mapping, Optional, Union
+
+from langchain.callbacks.manager import (
+    AsyncCallbackManagerForChainRun,
+    CallbackManagerForChainRun,
+)
+from langchain.load.serializable import Serializable
+from langchain.schema.runnable.base import Input, Output, Runnable
+from langchain.schema.runnable.config import RunnableConfig
+from langchain.schema.runnable.passthrough import RunnablePassthrough
+
+
+class PutLocalVar(RunnablePassthrough):
```

Comment:
Could we document scope semantics somewhere?

Reply:
as in what is meant by "Local" in this case?